tests/python/test_f16.py

```diff
@@ -51,6 +52,45 @@ def init():
         assert (z[i] == i * 3)
 
 
+@pytest.mark.skipif(not ti.has_pytorch(), reason='Pytorch not installed.')
+@ti.test(arch=archs_support_f16)
+def test_to_torch():
+    n = 16
+    x = ti.field(ti.f16, shape=(n))
```

Comment:
`ti.field` can accept a tuple or an int as its shape. However, `(n)` is an int which looks like a tuple. It might be clearer to make it more specific, i.e., `n` or `(n, )`.